djangocms_bootstrap4/contrib/bootstrap4_grid/cms_plugins.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
```

Comment:
add this at the top after the utf8

```from __future__ import unicode_literals```

Reply:
I know we had this talk once before, always on cmsplugins and models (generally where models are involved) or? Also on forms.py for example?